### DISCUSSION TITLE
Login fails after migrating updating to 3.6.6

### DISCUSSION THREAD

**Discussion by KosenkoIvan** [netbox-community/netbox#14474] [Help Wanted!]
![image](https://github.com/netbox-community/netbox/assets/97949308/05916678-29dd-4138-9191-d32b6041df5b)
After updating to 3.6.6 it is impossible to log into the admin account

**Comment by jeremystretch** [comment]
I want to say this may be related to bug #14249 (which has been fixed in the upcoming v3.6.7 release), but it's not clear how this is being triggered. Try enabling debug mode (set `DEBUG=True` in `configuration.py`) and posting the stack trace.

**Reply by KosenkoIvan** [reply]
Problem solved, incorrect password was entered. But the error window was :
<class 'netaddr.core.AddrFormatError'>
failed to detect a valid IP address from ''

**Comment by KosenkoIvan** [comment]
Problem solved, incorrect password was entered. But the error window was :
<class 'netaddr.core.AddrFormatError'>
failed to detect a valid IP address from ''
